azext_iot/operations/hub.py

```diff
@@ -1907,6 +1908,7 @@ def iot_get_sas_token(
     login=None,
     module_id=None,
     auth_type_dataplane=None,
+    connection_string=None,
```

Comment:
Would be good to add an IT to exercise the command with the connection string only option. You would run into an issue there because for iothub commands currently either `hub_name` or `login` are required. We need to also allow `connection_string` for this command. You can modify that validation [here](https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/_validators.py#L11).